apps/src/blockly/addons/contextMenu.js

```diff
@@ -148,21 +153,59 @@ const registerKeyboardNavigation = function () {
     },
     callback: function (scope) {
       const controller = Blockly.navigationController;
-      scope.workspace.keyboardAccessibilityMode
-        ? controller.disable(scope.workspace)
-        : controller.enable(scope.workspace);
+      if (scope.workspace.keyboardAccessibilityMode) {
+        controller.disable(scope.workspace);
+      } else {
+        controller.enable(scope.workspace);
+        Blockly.navigationController.navigation.focusWorkspace(scope.workspace);
+      }
     },
     scopeType: GoogleBlockly.ContextMenuRegistry.ScopeType.WORKSPACE,
     id: 'keyboardNavigation',
-    weight: 11,
+    weight,
   };
   GoogleBlockly.ContextMenuRegistry.registry.register(keyboardNavigationOption);
 };
 
+/**
+ * Change cursor type for keyboard navigation
+ */
+const registerCursor = function (cursorType, weight) {
```

Comment:
nit: Could we call this `registerCursorOptions`?

Reply:
It can be `registerCursorOption` (not plural). However, none of the other functions in this file use that convention. Possibly worth noting, we borrow the current convention from core Blockly: https://github.com/google/blockly/blob/develop/core/contextmenu_items.ts